Firebase/Auth/Source/Private/FirebaseAuth_Internal.h

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Is there a way to share the new _Internal.h/_Private files, since they're the same? At least they should be consistently named. The others are all _Private.h.